src/main/java/org/jenkinsci/test/acceptance/docker/fixtures/SshdContainer.java

```diff
@@ -63,6 +64,7 @@ public File getEncryptedPrivateKey() {
         return privateKeyEnc;
     }
 
+    @SuppressFBWarnings(value = "DM_DEFAULT_ENCODING", justification = "TODO needs triage")
```

Comment:
Actually it should be ASCII.

Reply:
Tbh, I didn't really look at what's left to do, just focused on Java 21 compatibility 👀